src/core/receiver/gnss_flowgraph.cc

```diff
@@ -1010,7 +1010,8 @@ int GNSSFlowgraph::connect_signal_conditioners_to_channels()
 
             try
                 {
-                    selected_signal_conditioner_ID = configuration_->property("Channel" + std::to_string(i) + ".RF_channel_ID", 0);
+                    selected_signal_conditioner_ID = configuration_->property("Channels_" + channels_.at(i)->get_signal().get_signal_str() + ".RF_channel_ID", 0);
+                    selected_signal_conditioner_ID = configuration_->property("Channel" + std::to_string(i) + ".RF_channel_ID", selected_signal_conditioner_ID);
```

Comment:
I don't know if there is any such documentation in the code, but the precedence implemented here should be documented somewhere, I think. Otherwise, this is a nice change.

Reply:
I agree. We should document a new option and the fact, that an old option overrides the new one. I'll fork https://github.com/gnss-sdr/geniuss-place and open a PR.